README.md

```diff
@@ -35,12 +35,15 @@ See License.txt in the project root for license information.
 Portions of this project are licensed under the MIT license.
 See MIT-License.txt in the project root for license information.
 
+The Release Package on Nuget.org
+================================
+The official release NuGet packages for Open XML SDK are available on Nuget.org at https://www.nuget.org/packages/DocumentFormat.OpenXml. 
 
 The Latest Builds
 =================
 
-## Where to get the NuGet package?
-The NuGet package for Open XML SDK is currently available as a custom feed on MyGet. You can trust this package source, since the custom feed is locked and only this project feeds into the source.
+## Where to get the latest build NuGet package?
```

Comment:
This reads a little weird - maybe "Where to get the NuGet packages of the latest builds"?